CMakeLists.txt

```diff
@@ -43,3 +40,17 @@ include_directories(${MPI_Fortran_INCLUDE_PATH})
 # Recursively build source tree
 add_subdirectory (@env)
 add_subdirectory (src)
+
+# https://www.scivision.dev/cmake-auto-gitignore-build-dir/
+# --- auto-ignore build directory
+if(NOT EXISTS ${PROJECT_BINARY_DIR}/.gitignore)
```

Comment:
It's a fixture, so this is probably safe, but `${GEOSldas_BINARY_DIR}` is generally safer.